arex-schedule-web-api/src/main/java/com/arextest/schedule/service/report/QueryReplayMsgService.java

```diff
@@ -0,0 +1,108 @@
+package com.arextest.schedule.service.report;
+
+import com.arextest.diff.model.enumeration.DiffResultCode;
+import com.arextest.diff.model.log.LogEntity;
+import com.arextest.diff.model.log.NodeEntity;
+import com.arextest.diff.model.log.UnmatchedPairEntity;
+import com.arextest.diff.sdk.CompareSDK;
+import com.arextest.schedule.dao.mongodb.ReplayCompareResultRepositoryImpl;
+import com.arextest.schedule.model.ReplayCompareResult;
+import com.arextest.schedule.model.converter.ReplayCompareResultConverter;
+import com.arextest.schedule.model.report.QueryDiffMsgByIdResponseType;
+import com.arextest.schedule.model.report.QueryLogEntityRequestTye;
+import com.arextest.schedule.model.report.QueryLogEntityResponseType;
+import com.arextest.schedule.utils.ListUtils;
+import com.arextest.schedule.model.report.CompareResultDetail;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang3.tuple.MutablePair;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.Resource;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+
+@Slf4j
+@Component
+public class QueryReplayMsgService {
+    @Resource
+    private ReplayCompareResultRepositoryImpl replayCompareResultRepository;
+    private static final CompareSDK COMPARE_INSTANCE = new CompareSDK();
```

Comment:
For the scattered use of CompareSDK, is it possible to add a method to the original DefaultReplayResultComparer?